docs/concepts/glossary.md

```diff
@@ -79,7 +79,7 @@ whole [Flink Master](#flink-master) was called JobManager.
 #### Logical Graph
 
 A logical graph is a directed graph describing the high-level logic of a stream processing program.
-The nodes are [Operators](#operator) and the edges indicate input/output-relationships or
```

Comment:
I think this sounded correct as it was.

Reply:
I am pretty sure it is a copy-paste typo from the Physical Graph where the wrong preposition got removed along with "partitions".
Physical graph:
"The nodes are Tasks and the edges indicate input/output-relationships **or** partitions **of** data streams or data sets." 

Also the construct "this OR that OR that" sounds awkward and people usually write "this, that OR that" when listing more than two items.

Reply:
I agree with Stephan. "... input/output-relationships of data streams or data sets" does not seem right to me. The edges describe the relationships of *operators* but not of *data streams/sets*. 

You're probably correct that this part was copy/pasted. In the original sentence, the words belong together as follows: "(input/output-relationships) or (partitions of data streams or data sets)". Since, a logical graph is not parallelized, the "partitions of" was removed. 

Although correct, I agree that "input/output-relationships or data streams or data sets." doesn't sound good and could be improved. How about:

> ... the edges indicate input/output-relationships of the operators and correspond to data streams or data sets.

Reply:
@fhueske changed accordingly and rebased on the current master.